scripts/uart_terminal/scripts/get_serial_ports.py

```diff
@@ -0,0 +1,37 @@
+"""
+#
+# Copyright (c) 2022 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+"""
+
+import sys
+import subprocess
+import re
+
+
+def get_serial_ports():
+    """Scan for all Nordic borads attached to a serial port method"""
```

Comment:
Nitpick: There is a newline after this line in the other similar methods

Reply:
Ok